File: Ejercicio2/BicicletaMontaña.py
from Bicicleta import Bicicleta

class BicicletaMontaña(Bicicleta):
    id_carrera = 0

    def __init__(self):
        super().__init__()

    def __str__(self):
        return f"Bicicleta de Montaña con identificador de carrera {self.get_id_carrera()}"
    

File: P1/Ejercicio1/BicicletaMontaña.java
package Ejercicio1;

public class BicicletaMontaña extends Bicicleta {
    private static int id = 0;

    public BicicletaMontaña() {
        super(id);
        ++id;
    }

    @Override
    public String toString() {
        return ("Bicicleta de montaña con identificador de carrera " + getIdCarrera());
    }
}
